### PR TITLE
[sdg] updates

### PR DESCRIPTION
**Pull request by chejennifer**
- update "m^s" and "km^2" to use superscripts
- fix color coding on map legend
- update a util function comment from https://github.com/datacommonsorg/website/pull/3585
Screenshot 2023-09-14 at 3 35 44 PM
Screenshot 2023-09-14 at 3 36 42 PM
Screenshot 2023-09-14 at 4 17 48 PM

### CODE REVIEW

File: static/js/utils/tile_utils.tsx [hunk 1]
```diff
@@ -376,6 +377,18 @@ export function getSourcesJsx(sources: Set<string>): JSX.Element {
   );
 }
 
+// Processes a unit string
```
Comment (juliawu) [inline static/js/utils/tile_utils.tsx:380]: Let's add a comment that this converts a units with "^" into superscript.

**Review by juliawu — APPROVED** [review]
Very nice!

File: static/js/utils/tile_utils.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/utils/tile_utils.tsx]: done

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**